src/wallet/wallet.cpp

```diff
@@ -1914,15 +1919,14 @@ void CWallet::ResendWalletTransactions()
     { // cs_wallet scope
         LOCK(cs_wallet);
 
-        // Relay transactions
-        for (std::pair<const uint256, CWalletTx>& item : mapWallet) {
-            CWalletTx& wtx = item.second;
+        // Try to add wallet transactions to memory pool
```

Comment:
"and relay"?

Reply:
you meant the variable? the change is on `void CWallet::ResendWalletTransactions()`.

Reply:
Done